postreise/plot/plot_utilization_map.py

```diff
@@ -35,92 +20,109 @@ def map_risk_bind(
     vmin=None,
     vmax=None,
     is_website=False,
+    palette=None,
+    all_branch_scale_factor=5e-4,
+    all_branch_min_width=0.2,
+    select_branch_scale_factor=1e-3,
+    select_branch_min_width=2,
+    figsize=(1400, 800),
+    show_color_bar=True,
+    plot_states_kwargs=None,
 ):
```

Comment:
I believe `us_states_dat=None` can be discarded

Reply:
Done.